docs/apache-airflow-providers-common-sql/supported-database-types.rst

```diff
@@ -0,0 +1,46 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+Supported Database Types
+========================
+
+The ``apache-airflow-providers-common-sql`` provider is compatible with the following
+database types:
+
+- :doc:`PostgreSQL <apache-airflow-providers-postgres:index>`
+- :doc:`Apache Impala <apache-airflow-providers-apache-impala:index>`
+- :doc:`Elastic Search <apache-airflow-providers-elasticsearch:index>`
+- :doc:`Microsoft SQL Server <apache-airflow-providers-microsoft-mssql:index>`
+- :doc:`Slack <apache-airflow-providers-slack:index>`
+- :doc:`Databricks <apache-airflow-providers-databricks:index>`
+- :doc:`Vertica <apache-airflow-providers-vertica:index>`
+- :doc:`JDBC <apache-airflow-providers-jdbc:index>`
+- :doc:`Apache Hive <apache-airflow-providers-apache-hive:index>`
+- :doc:`Apache Druid <apache-airflow-providers-apache-druid:index>`
+- :doc:`MySQL <apache-airflow-providers-mysql:index>`
+- :doc:`ODBC <apache-airflow-providers-odbc:index>`
+- :doc:`Snowflake <apache-airflow-providers-snowflake:index>`
+- :doc:`Amazon <apache-airflow-providers-amazon:index>`
+- :doc:`Presto <apache-airflow-providers-presto:index>`
+- :doc:`Trino <apache-airflow-providers-trino:index>`
+- :doc:`Apache Pinot <apache-airflow-providers-apache-pinot:index>`
+- :doc:`Pgvector <apache-airflow-providers-pgvector:index>`
+- :doc:`SQLite <apache-airflow-providers-sqlite:index>`
+- :doc:`Exasol <apache-airflow-providers-exasol:index>`
+- :doc:`Google <apache-airflow-providers-google:index>`
+- :doc:`Open Lineage <apache-airflow-providers-openlineage:index>`
+- :doc:`Apache Drill <apache-airflow-providers-apache-drill:index>`
```

Comment:
Can you sort this alphabetically for easier read ?

Reply:
Good call!

Reply:
That is a great improvement.